finish mobile version project

DamenaM opened the pull request:
# Request of  PR I  finish the mobile version project

- [ ] create Works section
- [ ] check  linter error
- [ ] Create About myself section
- [ ]  adding cards template using Flexbox to lay out 
- [ ] use CSS Grid and Flexbox

Olamarx commented on style.css:
```diff
@@ -101,3 +100,235 @@ body {
 }
+
+/* social media icons----------------------------------------------------------------------- */
+.Workssection {
+  border-radius: 8px;
+}
```
- [ ] For proper organizing of the content in this tag, could you consider styling it with grid as display property? You can read about the usefulness [here](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns)

Olamarx commented on index.html:
```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
 </section>
+<section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (1).png" alt=""></a>
```
- [ ] Kindly consider using this picture through CSS as a background image as decorations are always backgrounded images, not img tags.

Olamarx commented on index.html:
```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+
+    <div class="works2">
+        <div>html</div>
+        <div>css</div>
+        <div>javascript</div>
+   </div>
```
- [ ] Kindly consider using ul tag and each takes li tag to satisfy semantic purpose.

Olamarx commented on index.html:
```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
+        <div>javascript</div>
+   </div>
+   <div class="buttondir">
+    <button class="buttonProject"  type="button" class="resume-btn">See Project</button>
```
- [ ] Kindly consider the color as used on the figma
### Figma
![image](https://user-images.githubusercontent.com/86200356/199233248-4de7845f-bf84-421b-ab08-7b2b905fd3ea.png)

### Your Design
![image](https://user-images.githubusercontent.com/86200356/199233312-473a4136-79fc-4e41-a229-a20332ef60e9.png)

Olamarx commented on index.html:
```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
+    <div class="cardBack2">
+        <h3  class="tonic" >About Myself </h3>
+        <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
```
- [ ] This appears not to be on the Figma design. I would recommend sticking to the Figma template.

### Your Design
![image](https://user-images.githubusercontent.com/86200356/199234199-4580eecd-eada-4db9-8817-aee13d3ac8d4.png)

### Figma Design
![image](https://user-images.githubusercontent.com/86200356/199234386-f099390c-ce2f-4b0e-97b3-f95705c38a24.png)

Olamarx commented on index.html:
```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
+        and have a project you need coded, don’t hestiate to contact me. </p>
+</div>
+<div class = "social-section">
+    <h4 class="connectmywork"> LET’S CONNECT</p> 
```
- [ ] This confuses me. It is opening the tag with h4 and classes it with p tag 😒.

Olamarx reviewed (changes requested):
Hi @DamenaM, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media3.giphy.com/media/hXz6GN0ZilLwhgLo49/giphy.gif"/>

## Highlights
- All linters passed ✔️ 
- Good commit messages ✅ 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please consider applying the comment about button to all the buttons.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

DamenaM commented:
> based on PR comment

- [ ] border-radius
- [ ] I have changed my background image to CSS 
- [ ] I have changed div tag ul tag 
- [ ] I Have changed button color to figma design

sajid-munawar commented on index.html:
```diff
@@ -8,29 +8,173 @@
+<div class="buttonsee">
+    <button class="button1"> Get my resume </button>
+</div>
+        <ul class="Program-skil">
+            <li  class="Programs-lang" href="#"><img  src="./image/Frame 3.png" alt=""></li>
+            <ul class="Program-skil">
+                <li  class="Programs" href="#"><img src="./image/Frame 9.png" alt=""></li>
+                  <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
+                  <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
+              </ul>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
+        </ul>
```
- [Optional] Iam sorry that it was not pointed out by the previous reviewer, I strongly suggest you to change the languages from image to text as it is not an image. they are texts.
![img](https://user-images.githubusercontent.com/66992901/199460867-3a78b762-5e56-47e9-97a0-47be2b7dd7c3.PNG)

sajid-munawar reviewed (changes requested):
Hi @DamenaM , 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
